src/ports/postgres/modules/graph/measures.sql_in

```diff
@@ -0,0 +1,820 @@
+
+/**
+@addtogroup grp_graph_avg_path_length
+@brief Computes the average shortest-path length of a graph.
+
+<div class="toc"><b>Contents</b>
+<ul>
+<li><a href="#avg_path_length">Average Path Length</a></li>
+<li><a href="#examples">Examples</a></li>
+</ul>
+</div>
+
+This function computes the average of the shortest paths between each pair of
+vertices. Average path length is based on "reachable target vertices", so it
+ignores infinite-length paths between vertices that are not connected.
+
+@note This function assumes a valid output from a prior APSP run - both the APSP
+table and the associated output summary table. APSP is a computationally
+expensive algorithm because it finds the shortest path between all nodes in the
+graph. The worst case run-time for this implementation is O(V^2 * E) where V is
+the number of vertices and E is the number of edges.  In practice, run-time will
+be generally be much less than this, depending on the graph.
+
+@anchor avg_path_length
+@par Average Path Length
+<pre class="syntax">
+graph_avg_path_length( apsp_table,
+                       output_table
+                     )
+</pre>
+
+\b Arguments
+<dl class="arglist">
+<dt>apsp_table</dt>
+<dd>TEXT. Name of the output table generated by a prior run of all pairs
+shortest path (APSP).
+</dd>
+
+<dt>out_table</dt>
+<dd>TEXT. Name of the table to store the average path length. It contains a row
+for every group, and the average path value.
+</dd>
+</dl>
+
+@anchor examples
+@examp
+
+-# Create vertex and edge tables to represent the graph:
+<pre class="syntax">
+DROP TABLE IF EXISTS vertex, edge;
+CREATE TABLE vertex(
+        id INTEGER,
+        name TEXT
+        );
+CREATE TABLE edge(
+        src_id INTEGER,
+        dest_id INTEGER,
+        edge_weight FLOAT8
+        );
+INSERT INTO vertex VALUES
+(0, 'A'),
+(1, 'B'),
+(2, 'C'),
+(3, 'D'),
+(4, 'E'),
+(5, 'F'),
+(6, 'G'),
+(7, 'H');
+INSERT INTO edge VALUES
+(0, 1, 1.0),
+(0, 2, 1.0),
+(0, 4, 10.0),
+(1, 2, 2.0),
+(1, 3, 10.0),
+(2, 3, 1.0),
+(2, 5, 1.0),
+(2, 6, 3.0),
+(3, 0, 1.0),
+(4, 0, -2.0),
+(5, 6, 1.0),
+(6, 7, 1.0);
+</pre>
+
+-# Calculate the all-pair shortest paths:
+<pre class="syntax">
+DROP TABLE IF EXISTS out_apsp, out_apsp_summary;
+SELECT madlib.graph_apsp('vertex',      -- Vertex table
+                         'id',          -- Vertix id column (NULL means use default naming)
+                         'edge',        -- Edge table
+                         'src=src_id, dest=dest_id, weight=edge_weight',
+                                        -- Edge arguments (NULL means use default naming)
+                         'out_apsp');        -- Output table of shortest paths
+</pre>
+
+-# Compute the average path length measure:
+<pre class="syntax">
+DROP TABLE IF EXISTS out_avg_path_length;
+SELECT madlib.graph_avg_path_length('out_apsp', 'out_avg_path_length');
+SELECT * FROM out_avg_path_length;
+</pre>
+<pre class="result">
+ avg_path_length
+------------------
```

Comment:
This causes the result block separated into two halves. I tried `\------------------` and that seems to fix the issue. It does not happen with any other result blocks